### ISSUE TITLE
Add ability to paste

### ISSUE THREAD

**Issue by pwcazenave** [mtlynch/tinypilot#111]
It'd be handy to be able to paste in the contents of the clipboard into the remote machine (this is particularly useful for long, strong passwords which are a nuisance to type in).

**Comment by somik123** [comment]
+1 to this. Would be useful when we need to copy-paste linux commands as well for installation or configurations.
Maybe add a button that'll send whatever text is in the clipboard to the remote machine when clicked?

**Comment by pwcazenave** [comment]
As it happens, I had a stab at implementing this just after I wrote the issue.

I'm not very good at javascript, but I managed to hack something together which seems to work for me. One of the main issues I feel like I'm not handling all that well is I have a hard-coded list of modifier keys. This obviously doesn't support keyboard layouts beyond what I'm using.

In any case, I've opened a pull request #113 with what I put together.
